lib/game.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+require_relative 'item'
+
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(last_played_at, multiplayer)
+    super(last_played_at)
+    @last_played_at = last_played_at
```

Comment:
There seems to be a redundancy in initialization one in super and one in the sub class, wouldn't it be enough to use the super initialization?